### PR TITLE
Plone 5 compatibility

### PR DESCRIPTION
**Pull request by gyst**
This work builds on https://github.com/erral/ftw.blueprints and
- makes ftw.blueprints both Plone4 *and* Plone5 compatible
- removes hard ftw.inflator dependency so https://github.com/4teamwork/ftw.inflator/issues/29 does not block the use of ftw.blueprints

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
           'collective.blueprint.jsonmigrator',
           'collective.transmogrifier >= 1.5',
           'transmogrify.dexterity',
-          'ftw.inflator',
+          # 'ftw.inflator',
```
Comment (lukasgraf) [inline setup.py]: Remove it entirely instead of commenting out, IMHO.

**Comment by lukasgraf** [comment]
This mostly looks good to me 👍 

Just a couple minor things:
- Could you please add a `test-plone-5.0.x.cfg` (equivalent to the existing `test-plone-4.3.x.cfg`) so that the package gets tested on our CI against Plone 5?
- Remove the dependency on `ftw.inflator` entirely instead of commenting it out.

@jone with those changes I'd be ok with merging this, what do you think?

**Comment by jone** [comment]
👍 for removing the `ftw.inflator` dependency.
`ftw.inflator` is only used as example in the transmogrifier config. This is not really a dependency, so I'm in favor of removing it.

**Comment by lukasgraf** [comment]
Excellent, thank you! I couldn't quite get the tests to run locally using the `test-plone-5.0.x.cfg` yet . I expect the tests against `master` to fail after this merge, so I might remove it again for the time being.

*(Sorry for causing you unnecessary work, I assumed our CI would run tests against your PR, but that doesn't work yet because the fork lives in your GitHub account).*

**Comment by gyst** [comment]
OK please keep me updated. I'll need this soon for an ongoing project and would rather avoid the hassle of cooking my own forked egg for that...

**Comment by lukasgraf** [comment]
@gyst will do - just managed to get the Plone 5 tests working, and will cut a new release later today

**Comment by lukasgraf** [comment]
@gyst @erral 
I just released [`ftw.blueprints == 1.1.0`](https://pypi.python.org/pypi/ftw.blueprints/1.1.0) with Plone 5 compatibility 🎉  

Thank you both for your contributions! 🍰

**Pull request merged**